recipe/meta.yaml

```diff
@@ -36,4 +36,4 @@ about:
 extra:
   recipe-maintainers:
     - SylvainCorlay
-    - jmabille
+    - JohanMabille
```

Comment:
Is it ok with you to be a maintainer of `xtensor` in [conda-forge]( http://conda-forge.github.io/ ), @JohanMabille ?